### ISSUE TITLE
Parsing error: Expecting Unicode escape sequence \uXXXX

### ISSUE THREAD

**Issue by StigVanbrabant** [vuejs/eslint-plugin-vue#425]
**Tell us about your environment**

* **ESLint Version:** 4.15.0
* **eslint-plugin-vue Version:** 4.0.0
* **Node Version:** 8.3.0

**Please show your full configuration:**

```
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
  },
  // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
  // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
  extends: ['plugin:vue/essential', 'airbnb-base'],
  // required to lint *.vue files
  plugins: [
    'vue',
    'html'
  ],
  // check if imports actually resolve
  settings: {
    'import/resolver': {
      webpack: {
        config: 'build/webpack.base.conf.js'
      }
    },
    'html/indent': '+2',
    'html/report-bad-indent': 'error',
  },
  // add your custom rules here
  rules: {
    // don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      js: 'never',
      vue: 'never'
    }],
    // disallow reassignment of function parameters
    // disallow parameter object manipulation except for specific exclusions
    'no-param-reassign': ['error', {
      props: true,
      ignorePropertyModificationsFor: [
        'state', // for vuex state
        'acc', // for reduce accumulators
        'e' // for e.returnvalue
      ]
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      optionalDependencies: ['test/unit/index.js']
    }],
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-shadow': 0,
    'no-underscore-dangle': 0,
    'no-param-reassign': 0,
    'no-restricted-syntax': 0,
    'linebreak-style': 0,
    'o-absolute-path': 0,
    'no-new': 0,
    'no-console': 0,
    'no-bitwise': 0,
    'no-useless-escape': 0,
    'no-octal-escape': 0,
    'global-require': 0,
    'import/no-unresolved': 0,
    'no-param-reassign': 0,
    'valid-jsdoc': ['error', {
      'requireReturn': true,
      'requireReturnType': true,
      'requireParamDescription': true,
      'requireReturnDescription': true
    }],
    'prefer-destructuring': ['error', {
      'VariableDeclarator': {
        'array': false,
        'object': false
      },
      'AssignmentExpression': {
        'array': false,
        'object': false
      }
    }, {
      'enforceForRenamedProperties': false
    }],
    'require-jsdoc': ['error', {
        'require': {
            'FunctionDeclaration': true,
            'MethodDefinition': true,
            'ClassDeclaration': true
        }
    }],
    'max-len': ['error', {
      'code': 150
    }]
  }
}



```

**What did you do? Please include the actual source code causing the issue.**

```vue


const script = '<script type="text/javascript" src="..."><\/script>\n<noscript><img src="..." width="1" height="1"></noscript>';


```

**What did you expect to happen?**

For the linter to succeed because it all works when i remove the vue plugin from the configuration file. And can't find a way the make it work with the plugin active.

**What actually happened? Please include the actual, raw output from ESLint.**

✘ http://eslint.org/docs/rules Parsing error: Expecting Unicode escape sequence \uXXXX

**Comment by StigVanbrabant** [comment]
Found a workaround by putting it in the webpack environment files.

**Comment by dietergeerts** [comment]
@StigVanbrabant , what do you mean with this? I know webpack, but never heard of environment files.

**Comment by StigVanbrabant** [comment]
@dietergeerts When you generate a project with the vue webpack generator for example there is a config folder where you can assign variables that wel be exposed in your application as process.env, i put the string that caused the error inside there en then used it via the process.env

**Comment by dietergeerts** [comment]
Thx for explaining.
